doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -142,7 +142,16 @@ Bluetooth mesh samples
 nRF9160 samples
 ---------------
 
-|no_changes_yet_note|
+* :ref:`nrf_cloud_rest_cell_pos_sample` sample:
+
+  * Added:
+
+    * Use GCI search option if running modem firmware 1.3.4+.
```

Comment:
```suggestion
    * Usage of GCI search option if running modem firmware 1.3.4+.
```
What does 1.3.4+ actually mean? Is it v1.3.4 and later?

Reply:
Yes, probably not necessary for this change log though?  Stating 1.3.4 should be sufficient.